MapboxNavigation/Style.swift

```diff
@@ -31,14 +31,9 @@ open class Style: NSObject {
     @objc public var styleType: StyleType = .day
     
     /**
-     URL of the style to display on the map during turn-by-turn navigation.
+     URL of the style to display on the map during turn-by-turn navigation and on the map when previewing a route, for example on CarPlay or your own route preview map.
      */
-    @objc open var mapStyleURL: URL = MGLStyle.navigationGuidanceDayStyleURL
-    
-    /**
-     URL of the style to display on the map when previewing a route, for example on CarPlay or your own route preview map.
-     */
-    @objc open var previewMapStyleURL: URL = MGLStyle.navigationPreviewDayStyleURL
```

Comment:
I wonder if developers need the flexibility to specify unrelated styles for previewing versus guidance, particularly in the context of CarPlay where the navigation SDK manages the map style. If they do need that flexibility, then `MGLMapView.showsTraffic` would be insufficient; we could keep this property around but have it default to the same value as `mapStyleURL`.